packages/api-auth-lib/package.json

```diff
@@ -10,6 +10,7 @@
   "dependencies": {
     "express": "^4.16.3",
     "moment": "^2.24.0",
+    "node-cleanup": "^2.1.2",
```

Comment:
I'd rather not introduce a dep that is basically unmaintained (last commit 2017). Do we need it? Could we do what we need ourselves? See https://github.com/jtlapp/node-cleanup/issues/16

Reply:
I switched to doing manual cleanup and removed the package.